src/Components/Collections/CollectionsUtils/InputUtils/AddStr.js

```diff
@@ -20,7 +20,7 @@ const AddStr = ({identifier, ...props}) => {
         <TextField
             name="value"
             type="text"
-            value={v[0]=='\''?v.trim().slice(1, -1):v}
+            value={v[0]=='"'?v.trim().slice(1, -1):v}
```

Comment:
Same as previous comment, I think this might result in unexpected behavior for the user. Is there a valid reason to not just accept the user input without touching the surrounded quotes?